### PR TITLE
update ElasticJob.en

### LINKED ISSUE #6070: [Translation] Translate technical articles concerning Apache ShardingSphere
Hi, community,

This issue is to call for volunteers to **translate some technical articles from Chinese into English**. Anyone who enjoys translation or wants to make some contributions to open source community is warmly welcomed! 👏 

### Competencies needed
1. English translation skill
2. Basically, know what ShardingSphere is doing

### Article list
- [x] 1. [Start ElasticJob off with new goal](https://mp.weixin.qq.com/s/XJFj_vOj3bR6HRQUzy1ikg) --> @rudy2steiner
- [x] 2. [How to construct the distributed database](https://mp.weixin.qq.com/s/YSoX0Apq6vQeu8TU_53EAw) --> @EasonFeng5870
- [x] 3. [Apache ShardingSphere is included in CNCF Landscape](https://mp.weixin.qq.com/s/hAxON1CMYvgFf6sFIHtDHA) --> @mlw12345678
- [x] 4. [The mixed open-source distributed transaction solution](https://mp.weixin.qq.com/s/fYESyuw7O4u7wB-tSWlhyg) --> @mlw12345678
- [x] 5. [Would you like to become an Apache committer](https://mp.weixin.qq.com/s/sFqOz10KTk23IhmmDTy67g) --> @mlw12345678
- [x] 6. [How automatic executor of ShardingSphere works](https://mp.weixin.qq.com/s/hIKZPJDfGfeduW4frwVN_w)
- [x] 7. [How to merge the child resultsets](https://mp.weixin.qq.com/s/5__SGXEdvGqkVhpnrW6zug)
- [ ] 8. [The quick explanation of ShardingSphere transaction module](https://mp.weixin.qq.com/s/0LtqRTL8HewUAdZCk8LRLg)
- [x] 9. [What is the ShardingProxy](https://mp.weixin.qq.com/s/bTX0n9tH4x8eue19b7rdGw) --> @wcsoft 

### How to submit your translation
1. Add your translation to `xxx.en.md` in [blog](https://github.com/apache/shardingsphere/tree/master/docs/blog/content/material).

2. Raise a PR for review.

### Notice
1. All the Chinese articles named `xxx.cn.md` could be found in the same directory of Your translation as well. 
2. If you want o find a partner to work together for these long articles, please be free to give `recruit comment` below. 😉 
3. You can give a brief translation for these author profiles in the articles. Like just translate their names and titles.

If you want to have a try, **please comment here with** `Article number and your estimated deadline`. 

Besides, **any feedback and comments are welcomed as well**! 😃

### PR DESCRIPTION
**Pull request by rudy2steiner**
Fixes #6070 .

Changes proposed in this pull request:
- Keep Chinese  with English  version at the same time for compare and review 
- Introduce key features  with a feature list for readable
- Add reference

### CODE REVIEW

File: docs/blog/content/material/ElasticJob.en.md [hunk 1]
```diff
@@ -1,26 +1,37 @@
...
 
-> Discussion Email：
-https://lists.apache.org/thread.html/rd6171e2065be6bcfbeb7aba7e5c876eeed04db585c6ab78fc03a581c%40%3Cdev.shardingsphere.apache.org%3E
+<!-- 近两年来，由于核心开发者全力发展和维护Apache ShardingSphere项目等原因，ElasticJob放缓了发展的脚步。但是随着Apache ShardingSphere项目在分布式领域的不断拓展，以及用户对于多数据集群环境的分布式管理和数据迁移的弹性扩容等功能，需要一个强大的分布式调度组件。由此为契机，Apache ShardingSphere社区已经计划发起提议将ElasticJob作为其子项目进行长期持续的更新和维护。 -->
+In the last two years, the core developer spent much time to maintain and contribute to Apache ShardingSphere have caused ElasticJob development slowly.But with the developement of Apache ShardingSphere on distributed domain,we need a distributed scheduling component to support distributed management of multi cluster,data migration and scaling. So Apache ShardingSphere community have put up a proposal,make ElasticJob as a sub project,that makes sure ElasticJob could be maintain and update in the future.
```
Comment (tristaZero) [inline docs/blog/content/material/ElasticJob.en.md:27]: > the core developer **spent** much time to maintain and contribute to Apache ShardingSphere **have** caused ElasticJob development slowly.
> So Apache ShardingSphere community have put up a proposal,**make** ElasticJob as a sub project,that makes sure ElasticJob could be **maintain** and **update** in the future.

Hi @rudy2steiner Thanks for your translation and welcome!
I just gave it a glance and found some bad grammar, which made those sentences hard understood. So IMO, it is needed to check the grammar and word spelling with the help of APPs, like MS word.

Comment (rudy2steiner) [reply docs/blog/content/material/ElasticJob.en.md]: Thanks, any more Grammarly issue?

**Comment by tristaZero** [comment]
Hi @rudy2steiner Thanks for your prompt feedback. 
Although I merged this one, I am sorry to say it is not a perfect one, which means more polish is required.
For instance,

Do you think the grammar of this sentence is correct?
> In the last two years, the core developer spent much time maintaining and contributing to Apache ShardingSphere caused ElasticJob development slowly.

At least, it should be,
> In the last two years, the core developers spent much time maintaining and contributing to Apache ShardingSphere, **which** caused ElasticJob to develop slowly (or **caused slow development**).

**Comment by rudy2steiner** [comment]
Sorry, i will fix it by a new PR  if you not mind

**Pull request merged**